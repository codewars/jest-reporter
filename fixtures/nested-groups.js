describe("group 1", () => {
  describe("group 1 1", () => {
    test("test 1", () => {
      expect(1).toBe(1);
    });
  });
});
