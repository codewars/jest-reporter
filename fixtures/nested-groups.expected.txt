
<DESCRIBE::>group 1

<DESCRIBE::>group 1 1

<IT::>test 1

<PASSED::>Test Passed

<COMPLETEDIN::>1

<COMPLETEDIN::>

<COMPLETEDIN::>
